tests/extract/test_extract_job.sql

```diff
@@ -99,7 +100,41 @@ FROM
+        FROM
+            (SELECT 0) AS `t0`
+                LEFT JOIN
+            `test_treeherder`.`job` AS `t1` ON `t1`.`id` = `t0`.`id`
+                LEFT JOIN
+            `test_treeherder`.`build_platform` AS `t2` ON `t2`.`id` = `t1`.`build_platform_id`
+                LEFT JOIN
+            `test_treeherder`.`failure_classification` AS `t3` ON `t3`.`id` = `t1`.`failure_classification_id`
+                LEFT JOIN
+            `test_treeherder`.`job_group` AS `t4` ON `t4`.`id` = `t1`.`job_group_id`
+                LEFT JOIN
+            `test_treeherder`.`job_type` AS `t5` ON `t5`.`id` = `t1`.`job_type_id`
+                LEFT JOIN
+            `test_treeherder`.`machine` AS `t6` ON `t6`.`id` = `t1`.`machine_id`
+                LEFT JOIN
+            `test_treeherder`.`machine_platform` AS `t7` ON `t7`.`id` = `t1`.`machine_platform_id`
+                LEFT JOIN
+            `test_treeherder`.`option_collection` AS `t8` ON `t8`.`option_collection_hash` = `t1`.`option_collection_hash`
+                LEFT JOIN
+            `test_treeherder`.`product` AS `t9` ON `t9`.`id` = `t1`.`product_id`
+                LEFT JOIN
+            `test_treeherder`.`push` AS `t10` ON `t10`.`id` = `t1`.`push_id`
+                LEFT JOIN
+            `test_treeherder`.`repository` AS `t11` ON `t11`.`id` = `t1`.`repository_id`
+                LEFT JOIN
+            `test_treeherder`.`reference_data_signatures` AS `t12` ON `t12`.`id` = `t1`.`signature_id`
+                LEFT JOIN
+            `test_treeherder`.`option` AS `t13` ON `t13`.`id` = `t8`.`option_id`
+                JOIN
+            `test_treeherder`.`commit` AS `t14` ON `t14`.`push_id` = `t10`.`id`
```

Comment:
Here is the join to pull the `commit` data. This test exposes the large query used to pull those commit records: certainly it can be optimized; which we will do later.